cw/cw305/capture_traces.py

```diff
@@ -14,13 +14,20 @@
 
 from util import device
 from util import plot
+from util import spiflash
 
 
-def initialize_test(device_cfg):
+def initialize_test(device_cfg, spiflash_cfg):
```

Comment:
Maybe this should be called `initialize_capture` for the sake of clarity (+ the docstring below).

Reply:
Done.